pages/archiving/setup.rst

```diff
@@ -56,6 +56,127 @@ The archived indices will be stored in a backend. A backend that stores the data
 when the server starts for the first time but you can create a new backend if you want to store the data in a different
 path.
 
+
+S3 Archiving
+~~~~~~~~~~~~
+
+Graylog provides archiving built to be used against AWS S3. This doesn't mean it's limited
+to the AWS solution. You can configure S3 compatible solutions like MinIO, CEPH, Digital Ocean 
+Spaces and others.
```

Comment:
```suggestion
The S3 Archiving backend can be used to upload archives to an AWS S3 object storage service.
It is built to work with AWS, but should be compatible with other S3 implementations like MinIO, CEPH, Digital Ocean 
Spaces, etc.
```

Reply:
Yeah it's a little more verbose, but I lean toward Oxford comma. Regardless, I like your suggestion. So I'll keep it. Thx